modules/ossm-cert-manager-integration-istio.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration-istio_{context}"]
+== Integrating Service Mesh with cert-manager and istio-csr
+
+cert-manager is a solution for X.509 certificates management on Kubernetes. It delivers unified API to integrate applications with private or public PKIs, like Vault, Google Cloud CAS, Let's Encrypt etc. 
+
+It obtains certificates from a variety of Issuers, both popular public Issuers as well as private Issuers, and ensures the certificates are valid and up-to-date. It also attempts to renew certificates at a configured time before expiry. 
+
+cert-manager also provides integration for Istio users. `istio-csr` is a CA server that handles CSR requests from Istio proxies and delegates signing to cert-manager that forwards CSRs to the configured CA server.
+
+[NOTE]
+====
+Red Hat provides support for integrating with `istio-csr` and cert-manager. It does not provide support for the `istio-csr` or cert-manager components. The use of community cert-manager and `istio-csr` shown here are for demonstration purposes only.
+====
+
+.Prerequisites
+* One of these versions of cert-manager: 
+** cert-manager v1.11+ 
+** community cert-manager operator v1.11+ 
+** cert-manager Operator for Red Hat OpenShift v1.10+
+
```

Comment:
```suggestion
* istio-csr v0.6.0+
```

Reply:
According to this https://github.com/maistra/istio-operator/tree/maistra-2.4/deploy/examples/cert-manager/istio-csr , Kiali is now a prerequisite. Does that need to be added here as well?